CONTRIBUTING.md

```diff
@@ -64,6 +64,7 @@ We capitalize the following terms:
 * Ansible
 * Candlepin
 * Chef
+* DEB
```

Comment:
Is there a source for this? I see "Deb" used frequently.

Reply:
I mean anywhere. For example, [debian.org](https://www.debian.org/). Does their documentation use "DEB"?

Reply:
https://www.debian.org/doc/manuals/debian-handbook/apt.en.html at least uses APT. https://en.wikipedia.org/wiki/Deb_(file_format) uses `deb`, all lower case.